static/docs/commands-reference/config.md

```diff
@@ -1,38 +1,42 @@
 # config
 
-Get or set per repository or global options.
-
-You can query/set/replace/unset options with this command. It takes a name - is
-a section and a key separated by a dot. This command reads and overwrites the
-DVC config file `.dvc/config` (if `--local` option is specified
-`.dvc/config.local` is modified instead).
-
-If option value is not provided and `--unset` is not specified this command
-returns current value of the option.
+Get or set repository or global DVC settings.
 
 ```usage
-    usage: dvc config [-h] [-q | -v] [--global] [--system] [--local] [-u] name
-                      [value]
+    usage: dvc config [-h] [-q | -v] [--global] [--system] [--local]
+                      [-u] name [value]
 
     positional arguments:
-        name                  Option name
-        value                 Option value
+        name                  Setting name
```

Comment:
we should make a PR to DVC to change these descriptions

Reply:
I did (: https://github.com/iterative/dvc/pull/1999

Reply:
Left a comment in iterative/dvc#1999

Reply:
Thanks Ruslan. From that comment I get the sense that we don't want to change this terminology in the command output, so I can figure out some other way to make a clear distinction between the command options (`--unset`, etc) and the DVC options (`core.remote`, etc). Please confirm 😃 (and feel fee to close https://github.com/iterative/dvc/pull/1999)